common/src/main/java/io/netty5/util/internal/logging/InternalLoggerFactory.java

```diff
@@ -154,5 +156,5 @@ public static InternalLogger getInstance(String name) {
     /**
      * Creates a new logger instance with the specified name.
      */
-    protected abstract InternalLogger newInstance(String name);
+    public abstract InternalLogger newInstance(String name);
```

Comment:
why we need this change ?

Reply:
Ah, leftover from something I tried. It's not needed.